p4rt_app/p4runtime/p4runtime_impl.cc

```diff
@@ -96,29 +104,26 @@ absl::Status AllowRoleAccessToTable(const std::string& role_name,
 
 sonic::AppDbTableType GetAppDbTableType(
     const pdpi::IrTableEntry& ir_table_entry) {
+
   // By default we assume and AppDb P4RT entry.
   return sonic::AppDbTableType::P4RT;
 }
 
-// Read P4Runtime table entries out of the AppDb, and append them to the read
-// response.
+// Read P4Runtime table entries out of the AppStateDb, and append them to the
```

Comment:
Isnt this APPDB ?

Reply:
We didn't do any of the notification changes upstream. So at this checkpoint it's still the AppStateDb.